bug_report.php

```diff
@@ -74,12 +75,29 @@
 require_api( 'string_api.php' );
 require_api( 'user_api.php' );
 require_api( 'utility_api.php' );
+require_api( 'api_token_api.php' );
 
-form_security_validate( 'bug_report' );
+# Check if this is a usual submission via UI or scripted autosubmission
```

Comment:
I don't understand why we would change bug_report.php or reference the auto report concept for here.  This should be only a concept that is understood by the new SOAP API.